Source/JavaScriptCore/Configurations/libJavaScriptCore.xcconfig

```diff
@@ -0,0 +1,44 @@
+// Copyright (C) 2009-2023 Apple Inc. All rights reserved.
+//
+// Redistribution and use in source and binary forms, with or without
+// modification, are permitted provided that the following conditions
+// are met:
+// 1. Redistributions of source code must retain the above copyright
+//    notice, this list of conditions and the following disclaimer.
+// 2. Redistributions in binary form must reproduce the above copyright
+//    notice, this list of conditions and the following disclaimer in the
+//    documentation and/or other materials provided with the distribution.
+//
+// THIS SOFTWARE IS PROVIDED BY APPLE INC. ``AS IS'' AND ANY
+// EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+// IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+// PURPOSE ARE DISCLAIMED.  IN NO EVENT SHALL APPLE INC. OR
+// CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+// EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+// PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+// PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+// OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+// (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+// OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+#include "Base-JavaScriptCore.xcconfig"
+
+CLANG_OPTIMIZATION_PROFILE_FILE = $(BUILT_PRODUCTS_DIR)/DerivedSources/JavaScriptCore/JavaScriptCore.profdata;
+WK_COMPRESSED_OPTIMIZATION_PROFILE_FILE = $(WK_COMPRESSED_OPTIMIZATION_PROFILE_FILE_$(USE_INTERNAL_SDK));
+WK_COMPRESSED_OPTIMIZATION_PROFILE_FILE_YES = $(WK_WEBKITADDITIONS_HEADERS_FOLDER_PATH)/Profiling/JavaScriptCore.profdata.compressed;
+
+CLANG_USE_OPTIMIZATION_PROFILE = $(CLANG_USE_OPTIMIZATION_PROFILE_$(USE_INTERNAL_SDK)_$(CONFIGURATION)_$(WK_PLATFORM_NAME));
+CLANG_USE_OPTIMIZATION_PROFILE_YES_Release_macosx = YES;
+CLANG_USE_OPTIMIZATION_PROFILE_YES_Release_iphoneos = YES;
+CLANG_USE_OPTIMIZATION_PROFILE_YES_Production_macosx = YES;
+CLANG_USE_OPTIMIZATION_PROFILE_YES_Production_iphoneos = YES;
+
+CLANG_INSTRUMENT_FOR_OPTIMIZATION_PROFILING = $(CLANG_INSTRUMENT_FOR_OPTIMIZATION_PROFILING_$(ENABLE_LLVM_PROFILE_GENERATION));
+CLANG_INSTRUMENT_FOR_OPTIMIZATION_PROFILING_ENABLE_LLVM_PROFILE_GENERATION = YES;
+
+SKIP_INSTALL = $(WK_NOT_$(FORCE_TOOL_INSTALL));
```

Comment:
I wonder if this should be `NO`. When I experimented with creating a JavaScriptCore staticlib, it was multiple gigabytes in size. Assuming yours is similarly huge we should avoid ever installing it into the SDK.

Reply:
Yes, it's 1.8 GB.  I'm fine changing this to `NO`.  I used the above since it was the convention for the targets.

```
$ du -sh Release/libJavaScriptCore.a 
1.8G	Release/libJavaScriptCore.a
```

Reply:
Actually, I think you mean `SKIP_INSTALL` should be set to `YES`, not `NO`, to always be skipped?

Reply:
Yeah, you're right, make it `YES` :)

Reply:
Done with next PR update.